src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -101,6 +102,17 @@
 logger = getLogger(__name__)
 
 LOCAL_TESTING_SUPPORTED_FILE_FORMAT = ("JSON",)
+
+TIME_TRAVEL_OPTIONS_PARAMS_MAP = {
```

Comment:
this would be exposed as part of public api.
if you think this is just an internal implementation detail, we can make it private by prepending an underscore `_TIME_TRAVEL_OPTIONS_PARAMS_MAP`